validation/suggest.go

```diff
@@ -31,13 +31,26 @@ func showSuggestion(validationErrors validationErrors) {
 	if !HideSuggestion {
 		for t, errs := range groupErrors(validationErrors) {
 			fmt.Println(getSuggestionMessage(t))
-			for _, err := range errs {
-				fmt.Println(err.Suggestion())
+			suggestions := filterDuplicateSuggestions(errs)
```

Comment:
When would there be duplicate suggestions?

Reply:
We have duplicate suggestions printed if there is an unimplemented step is used multiple times.

Reply:
I just tried it, it is printing without the duplicate suggestions.

Reply:
In master the suggestions was added to only one error. The duplicate errors did not have the suggestions. But this doesn't work well for lsp diagnostics. We need the suggestions on all errors. Hence, there are changes in validate.go where we copy over the suggestions while duplicating the errors. 
After this change, we had to filter the duplicate suggestion while printing on the console.